README.md

```diff
@@ -117,9 +145,121 @@ Get all reports for the user:
 Get-PowerBIReport
 ```
 
+If you are a tenant administrator, you can view all reports in your tenant by using assigning `-Scope Organization`:
+
+```powershell
+Get-PowerBIReport -Scope Organization
+```
+
+### Get Dashboards
+
+Get dashboards for the user:
+
+```powershell
+Get-PowerBIDashboard
+```
+
+If you are a tenant administrator, you can view all dashboards in your tenant by adding `-Scope Organization`:
+
+```powershell
+Get-PowerBIDashboard -Scope Organization
+```
+
+### Get Tiles
+
+Get tiles within a dashboard:
+
+```powershell
+Get-PowerBITile -DashboardId 9a58d5e5-61bc-447c-86c4-e221128b1c99
+```
+
+### Get Imports
+
+Get Power BI imports:
+
+```powershell
+Get-PowerBIImport
+```
+
+### Create Report
+
+Create a report in Power BI by uploading a *.pbix file:
+
+```powershell
```

Comment:
Do you know why the text from here appears as italic in this review? The rendered markdown file looks fine.

Reply:
It's because of the *, I will need to escape